app/views/pages/unsubscribe.html.erb

```diff
@@ -0,0 +1,11 @@
+<% content_for :header, "Why did I receive this email?" %>
+<p>
+  <%= t('.you_received_an_email') %>
+    no-reply@email.prisonvisits.service.gov.uk
```

Comment:
This interpolation is naïve, because it assumes that the email address always goes at the end of the sentence, which isn't always valid. I don't know about Welsh specifically, but it doesn't work in German or Japanese. Generally speaking, one should never form sentences by concatenation, but only via placeholders, _viz._:

```erb
<%= t('.you_received_an_email', from: no-reply@email.prisonvisits.service.gov.uk) %>
```